public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
+    {
+      "src": "favicon.ico",
+      "sizes": "64x64 32x32 24x24 16x16",
+      "type": "image/x-icon"
+    },
+    {
+      "src": "logo192.png",
+      "type": "image/png",
+      "sizes": "192x192"
+    },
+    {
+      "src": "logo512.png",
+      "type": "image/png",
+      "sizes": "512x512"
+    }
+  ],
+  "start_url": ".",
+  "display": "standalone",
+  "theme_color": "#000000",
+  "background_color": "#ffffff"
```

Comment:
Please kindly remove the unnecessary files from the setup project
![3CA18F96-E59B-43BB-8442-59B53D1208F6_1_201_a](https://github.com/binodbhusal/final_capstone_frontend/assets/124040422/3ba5455e-911c-4de6-90b2-beaa00a37091)

Reply:
Done :ok_hand: